Formula/valkyrie.rb

```diff
@@ -14,8 +14,14 @@ class Valkyrie < Formula
   depends_on "valgrind"
 
   def install
+    inreplace "src/utils/vk_utils.h", "#include <iostream>",
+                                      "#include <iostream>\n#include <unistd.h>"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

😉